docs/guide/browser.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Browser mode | Guide
+---
+
+# Browser mode (experimental)
+
+This document provides information about the experimental browser mode feature in the vitest API, which allows you to run your tests in the browser natively, providing access to browser globals like window and document. This feature is currently under development, and APIs may change in the future.
+
+## Configuration
+
+To activate browser mode in your vitest configuration, you can use the `--browser` flag or set the browser field to true in your vitest configuration file. Here is an example configuration using browser field:
+
+```ts
+export default defineConfig({
+  test: {
+    browser: true,
+  }
+})
+```
+Browser Option Types:
+
+The browser option in vitest can be set to either a boolean or a string type. If
+set to `true`, the users can navigate to the provided link by vitest. You can also specify a browser by providing its name as a string. The available browser options are:
+- `firefox`
+- `chrome`
+- `edge`
+- `safari`
+
+Here's an example configuration setting chrome as the browser option:
+
+```ts
+export default defineConfig({
+  test: {
+    browser: 'chrome',
+  }
+})
+```
+
+## Cross-browser Testing:
+
+When you specify a browser name in the browser option, vitest will try to run
+the specified browser using [WebdriverIO](https://webdriver.io/), and then run the tests there. This
+feature makes cross-browser testing easy to use and configure in environments
+like CI.
```

Comment:
I find manually wrapped lines harder to read and to maintain compared to letting the editor handle line wrapping
```suggestion
When you specify a browser name in the browser option, vitest will try to run the specified browser using [WebdriverIO](https://webdriver.io/), and then run the tests there. This feature makes cross-browser testing easy to use and configure in environments like a CI.
```